src/app/pages/home/rolodex/rolodex.page.ts

```diff
@@ -5,8 +5,7 @@ import { AuthService } from 'src/app/services/auth/auth.service';
 import { UserService, User } from './../../../services/user/user.service';
 import { CardService, Card } from 'src/app/services/card/card.service';
 
-import { Observable } from 'rxjs';
```

Comment:
Why change the "Observable" import?

Reply:
We could directly use the Observable but I've found it useful to instead subscribe to it, print diagnostic logs, and assign the subscribed data to a local variable for the front-end (html).